src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -828,4 +828,21 @@ export class BrowsingContextImpl {
       },
     };
   }
+
+  async close(): Promise<void> {
+    try {
+      await this.#cdpTarget.cdpClient.sendCommand('Page.close');
+    } catch (error: any) {
+      // Swallow error that arise from the page being destroyed
+      // Example is navigating to faulty SSL certificate
```

Comment:
Is it feasible to add a test for this?

Reply:
We don't have our own server we can control at this time, we can open an issue and label `local-server` to them once we have the infastructure.

Reply:
I started with 2 items, you can add a third one there (maybe as a subitem of the second item)